src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MIN_VALUE = 1;
+    private static final int MAX_VALUE = 127;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int randomFigureIndex = random.nextInt(NUMBER_OF_FIGURES);
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = colorSupplier.getRandomColor();
```

Comment:
all figures will be of the same color?

Reply:
no, figures will have different colours.